### ISSUE TITLE
Make it available in a single page to save it in a reader (like Pocket)

### ISSUE THREAD

**Issue by aug-riedinger** [greggman/webgl-fundamentals#10]
When I read about this (in that case on Hacker News), I'm like "why not read about it when I have time". 
The process becomes: save to Pocket, and then read it step by step.

But I don't want to spam my Pocket readlist with every article, but rather would save it as a long story.

Cheers

**Comment by stefnotch** [comment]
You could try to download the entire repository and opening it with your browser. (index.html file: https://github.com/greggman/webgl-fundamentals/blob/master/index.html )

**Comment by greggman** [comment]
That's a good idea

Making the entire site a single page would cause some technical issues as having 50 or so interactive WebGL demos on a page will run into browser limits so some work arounds would be required.

**Comment by aug-riedinger** [comment]
Yes, but so that pocket can save it, it has to be available online. Anyways, thanks for suggestion
